docs/concepts/streaming.md

```
@@ -40,26 +40,79 @@ Each chunk is one progress notification. The complete accumulated text is also s
-@mesh.route(dependencies=["chat"])
-async def chat_endpoint(body: ChatRequest, chat: McpMeshTool = None) -> mesh.Stream[str]:
-    async for chunk in chat.stream(prompt=body.prompt):
-        yield chunk
-```
+Once a producer streams, two consumer code paths are available — and both work in **all three runtimes** (Python, TypeScript, Java) as of #854.
+
+**MCP-native (`proxy.stream(...)`)** — when a mesh agent depends on a streaming tool, calling `proxy.stream(...)` (or its language equivalent) returns an iterator of chunks. The framework subscribes to progress notifications on the open MCP connection and yields each one to the consumer.
+
+=== "Python"
+
+    ```python
```

markdownlint-cli2 (0.22.1)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)